src/v/kafka/server/datalake_throttle_manager.h

```diff
@@ -86,12 +87,16 @@ class datalake_throttle_manager
     get_producer_throttle(const std::optional<std::string_view>&);
 
     std::chrono::milliseconds calculate_throttle() const;
-
+    void on_disk_space_update();
```

Comment:
is this implemented?

Reply:
oh, i assumed this would be needed to update disk info non-zero shards. where does that happen then?